client/python/kserve-api/samples/requirements.txt

```diff
@@ -0,0 +1,3 @@
+protobuf==3.19.3
+tritonclient==2.22.3
```

Comment:
Shouldn't tritonclient be enough? It should have dependency of protobuf and grpcio

Reply:
You cannot run the examples without all of those packages